config/helm/appmesh-controller/test.yaml

```diff
@@ -136,6 +136,14 @@ stats:
 # Enable cert-manager
 enableCertManager: false
 
+# podDisruptionBudget for Appmesh controller
+pdb:
+  # PodDistruptionBudget is stable from kubernetes version 1.21
+  # By default it will be disabled
+  enabled: false 
+  # minAvailable pods for appmesh-controller
+  minAvailable: 1
```

Comment:
Are we going to support minAvailable only?

Reply:
Do we need both ? As i understand this can be percentage as well. so couldn't we have maxUnavailable achieved with minAvailable alone ?

Reply:
I was thinking about something similar to 
podDisruptionBudget: {}
and generate the manifest based on this

Reply:
Got it , makes sense